SearchActivity.py

```diff
@@ -241,7 +241,8 @@ def _list_tubes_error_cb(self, e):
         """ Log errors. """
         _logger.debug('Error: ListTubes() failed: %s' % (e))
 
-    def _new_tube_cb(self, tube_id, initiator, tube_type, service, params, state):
+    def _new_tube_cb(self, tube_id, initiator, tube_type, service, params,
```

Comment:
One of those cases where the line should be left unchanged.

Reply:
On the other hand, it does fix `./SearchActivity.py:244:80: E501 line too long (82 > 79 characters)' so I'm in favour of the change.